r2k.c

```diff
@@ -41,6 +46,7 @@ static struct device *r2k_dev_ph;
 static struct class *r2k_class;
 static struct cdev *r2k_dev;
 static dev_t devno;
+unsigned long stack_guard_gap = 256UL<<PAGE_SHIFT;
```

Comment:
I've actually removed this line as a whole, since (afaik) this symbol's exported by the kernel itself. Let me know if I'm wrong in regard of this assumption.